vagrant/test/lib/box_loader_test.rb

```diff
@@ -10,10 +10,24 @@ def test_load
     loader.load!
 
     assert loader.boxes
+    assert loader.boxes['centos7-katello-nightly']
     assert_equal 'centos7-katello-nightly', loader.boxes['centos7-katello-nightly']['name']
     assert_equal 'centos/7', loader.boxes['centos7-katello-nightly']['box_name']
   end
 
+  def test_load_with_exclude
+    excludes = { 'boxes' => { 'exclude' => ['fedora'] } }
+    Forklift::Settings.any_instance.stubs(:settings).returns(excludes)
+    locations = get_locations('00-fedora.yaml', '03-packaging.yaml')
+    loader = Forklift::BoxLoader.new(nil, locations)
+    loader.load!
+
+    assert loader.boxes
+    assert loader.boxes['rpm-packaging']
+    assert_equal 'rpm-packaging', loader.boxes['rpm-packaging']['name']
+    assert_equal 'fedora/30-cloud-base', loader.boxes['rpm-packaging']['box_name']
```

Comment:
this could also be a simple `assert` as all we care about is that it's set at all (which it wasn't before)

Reply:
@ehelms opinions here?

Reply:
This looks good.